pulp_ansible/app/viewsets.py

```diff
@@ -267,6 +585,55 @@ class AnsibleRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin):
     queryset = AnsibleRepository.objects.all()
     serializer_class = AnsibleRepositorySerializer
 
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:ansible.add_ansiblerepository",
+            },
+            {
+                "action": "destroy",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:ansible.delete_ansiblerepository",
+            },
+            {
+                "action": [
+                    "modify",
+                    "partial_update",
+                    "rebuild_metadata",
+                    "repair",
+                    "sign",
+                    "sync",
+                    "update",
+                ],
```

Comment:
We typically add a permission for each extra action that can be performed on a repository. https://github.com/pulp/pulp_file/blob/main/pulp_file/app/viewsets.py#L143-L170.  If this is too granular you can combine some actions into one permission, but I think it would be helpful having separate sync and sign permissions.